Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/SqlServer/Model/Building/SqlStorageTypeInformationProviderTest.cs

```diff
@@ -461,6 +465,48 @@ public void GetStorageType_PropertyDefinition_ForNullableValueTypes ()
           Is.TypeOf(typeof(AdvancedEnumConverter)));
     }
 
+    [Test]
```

Comment:
it would be great if you could copy those tests for nullable. I know, whitebox says it's not needed but the call chains are complex enough to foul up the works if one doesn't watch out for details.

Please also add tests for GetStorageType(type) with DateTime and DateTime? and GetStorageType(object) with DateTime in both provider variants since we want to explicitly document that the new services covers these overloads, too.